### DISCUSSION TITLE
Roms needed

### DISCUSSION THREAD

**Discussion by TesterOfDoom** [86Box/86Box#2253] [Q&A] [answered]
So I just installed 86Box, and im having trouble using it, I need the rooms for it, I have cloned it with Github desktop, yet it won't work, any help, please? IM reinstalling the newest ver, maybe I had installed an older ver without knowing?

**Comment by NTechV** [comment]
I think that you might need to use the code -> download zip instead of cloning it.

**Reply by TesterOfDoom** [reply]
ok, I will try that

**Reply by TesterOfDoom** [reply]
did that, now what else, i got 86ButManager working now, so what do i do next

**Comment by NTechV** [comment]
Then you have to go to the 86box directory and extract the roms to it.

**Comment by NTechV** [comment]
After you do that, set the 86box path in 86box manager to the directory..

**Comment by NTechV** [comment]
Finally, create a machine for 86box and start it.

**Reply by TesterOfDoom** [reply]
i will do

**Reply by TesterOfDoom** [reply]
this is gonna sound stupid, but how do I extract the ROMs?

**Reply by TesterOfDoom** [reply]
plz help

**Comment by NTechV** [comment, marked as answer]
Then right click the ROMs file, click the extract all button and extract the ROMs to your 86box directory (The directory where you extracted the 86box files.).

**Reply by TesterOfDoom** [reply]
ok, i did that, now i need help with the iso
